redi-connect-front/src/components/molecules/ReadContacts.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import { Content } from 'react-bulma-components'
+import { RedProfile } from '../../types/RedProfile'
+import { connect } from 'react-redux'
+import { RootState } from '../../redux/types'
+import { Caption } from '../atoms'
+
+interface Props {
+  profile: RedProfile
+  caption?: boolean
+}
+
+const ReadContacts = ({ profile, caption }: Props) => {
```

Comment:
Can we rename this to `ReadContactDetails`? 'Contacts' is more about 'contacts I've got saved in my phone' than 'contact details of a specific contact'